django/db/backends/sqlite3/operations.py

```diff
@@ -441,3 +441,9 @@ def on_conflict_suffix_sql(self, fields, on_conflict, update_fields, unique_fiel
 
     def force_group_by(self):
         return ["GROUP BY TRUE"] if Database.sqlite_version_info < (3, 39) else []
+
+    def format_json_path_numeric_index(self, num):
+        """
+        SQLite backend has special syntax for negative indices in JSON paths.
+        """
```

Comment:
Everything in this module is pretty much "special"/specific to SQLite so I'd say we can leave out the docstring

```suggestion
```